packages/frontend/src/features/scheduler/components/SchedulerForm.tsx

```diff
@@ -770,6 +805,69 @@ const SchedulerForm: FC<Props> = ({
                             </Stack>
                         )}
 
+                        {isDashboardTabsEnabled &&
+                            isDashboardTabsAvailable &&
+                            !isThresholdAlert && (
+                                <Stack spacing={10}>
+                                    <Input.Label>
+                                        Tabs
+                                        <Tooltip
+                                            withinPortal={true}
+                                            maw={400}
+                                            multiline
+                                            label="Select all tabs to include all tabs in the delivery. If you don't select this option, only selected tab will be included in the delivery."
+                                        >
+                                            <MantineIcon
+                                                icon={IconHelpCircle}
+                                                size="md"
+                                                display="inline"
+                                                color="gray"
+                                                style={{
+                                                    marginLeft: '4px',
+                                                    marginBottom: '-4px',
+                                                }}
+                                            />
+                                        </Tooltip>
+                                    </Input.Label>
+                                    <Checkbox
+                                        size="xs"
+                                        label="Include all tabs"
+                                        labelPosition="right"
+                                        checked={allTabsSelected}
```

Comment:
this should be enabled by default, right ?

Reply:
yes, but this is not done by setting a default value for "allTabsSelected", but instead by setting initial the form value "selectedTabs" to include all the tabs, then dynamically derive the value from this form value.